MonsterTrainModdingAPI/Managers/CustomTriggerManager.cs

```diff
@@ -3,65 +3,179 @@
 using System.Collections.Generic;
 using System.Text;
 using MonsterTrainModdingAPI.Enums.MTTriggers;
+using HarmonyLib;
+using System.Linq;
 using UnityEngine;
+using Unity;
 
 namespace MonsterTrainModdingAPI.Managers
 {
     public class CustomTriggerManager
     {
-        private const int TriggerGUIDOffset = 100;
-        private static int NumTriggers = Enum.GetNames(typeof(CharacterTriggerData.Trigger)).Length + TriggerGUIDOffset;
         /// <summary>
-        /// Gets a New GUID
+        /// 576+ reservation makes it extraordinarily unlikely of a conflict with MT
+        /// </summary>
+        private static int NumCharTriggers = 576;
```

Comment:
Out of curiosity, why 576 in particular?

Reply:
I wished to reserve 0-64 for the game, and 512 integers for custom because those were numbers... Really it could be anything greater than ~27 and still work. Its not like it really matters...